server/core/src/actors/v1_write.rs

```diff
@@ -1820,4 +1821,57 @@ impl QueryServerWriteV1 {
 
         idms_prox_write.commit().map(|()| pw)
     }
+
+    #[instrument(
+        level = "info",
+        skip_all,
+        fields(uuid = ?eventid)
+    )]
+    pub(crate) async fn handle_domain_show(
+        &self,
+        eventid: Uuid,
+    ) -> Result<ProtoDomainInfo, OperationError> {
+        trace!("Begin domain show event");
+        let ct = duration_from_epoch_now();
+        let mut idms_prox_write = self.idms.proxy_write(ct).await;
```

Comment:
Is this a write transaction because we just haven't made it a reader? Feels weird to be pulling info and having to start a wtxn

Reply:
It's because in the admin command handler we only have the write handle :)

Reply:
That'll do it 😄